contact.html

```diff
@@ -70,7 +70,8 @@ <h2 style="color:white;">Let's Get in Touch.</h2>
                                             <input type="text" name="subject" value="" placeholder="Subject" required>
                                         </div>
                                         <div class="form-group col-md-6 col-sm-6 co-xs-12">
-                                            <input type="text" name="phone" value="" placeholder="Phone" required>
+                                        	<input type="text" name="phone" pattern="[6-9]{1}[0-9]{9}" value="" placeholder="Phone"title="Phone number with 7-9 and remaing 9 digit with 0-9" required>
```

Comment:
the title should be a valid phone number starting from 6-9 and rest of the digits from0-9
Also, give space between placeholder and title attribute.